core/src/main/java/org/libdohj/params/LitecoinRegTestParams.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2013 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.libdohj.params;
+
+import org.bitcoinj.core.Block;
+import org.bitcoinj.core.StoredBlock;
+import org.bitcoinj.core.VerificationException;
+import org.bitcoinj.store.BlockStore;
+import org.bitcoinj.store.BlockStoreException;
+
+import java.math.BigInteger;
+
+import static com.google.common.base.Preconditions.checkState;
+
+/**
+ * Network parameters for the regression test mode of bitcoind in which all blocks are trivially solvable.
+ */
+public class LitecoinRegTestParams extends LitecoinTestNet3Params {
+    private static final BigInteger MAX_TARGET = new BigInteger("7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", 16);
+
+    public LitecoinRegTestParams() {
+        super();
+        // Difficulty adjustments are disabled for regtest.
+        // By setting the block interval for difficulty adjustments to Integer.MAX_VALUE we make sure difficulty never changes.
+        interval = Integer.MAX_VALUE;
+        maxTarget = MAX_TARGET;
+        subsidyDecreaseBlockCount = 150;
+        port = 19444;
+        id = ID_LITE_REGTEST;
+        packetMagic = 0xfabfb5da;
+    }
+
+    @Override
+    public boolean allowEmptyPeerChain() {
+        return true;
+    }
+
+    private static Block genesis;
+
+
+    /* Extract from Litecoin source code, definition of regtest params.
```

Comment:
I'm not sure it's a good idea to paste this in here; a link to the commit in Github (i.e. https://github.com/litecoin-project/litecoin/blob/edc66b374ea68107c721062152dd95e6aa037d53/src/chainparams.cpp ) would probably make more sense, as once we have this working the parameters shouldn't change unless the original changes (in which case the copied data is unhelpful).